examples/js/controls/DragControls.js

```diff
@@ -51,11 +58,12 @@ THREE.DragControls = function ( _objects, _camera, _domElement ) {
 	function onDocumentMouseMove( event ) {
 
 		event.preventDefault();
+                pointer = event.changedTouches ? event.changedTouches[0] : event;
```

Comment:
Please insert spaces between parenthesis like so:
```javascript
event.changedTouches[ 0 ]
```
Besides, it looks like `pointer` is an undeclared variable.